CONTRIBUTING.MD

```diff
@@ -0,0 +1,19 @@
+# Contributing
+
+Thank you for your interest in `tools-python`. The project is open-source software, and bug reports, suggestions, and most especially patches are welcome.
+
+## Issues
+
+`tools-python` has a [project page on GitHub](https://github.com/spdx/tools-python/) where you can [create an issue](https://github.com/spdx/tools-python/issues/new) to report a bug, make a suggestion, or propose a substantial change or improvement that you might like to make. You may also wish to contact the SPDX working group technical team through its mailing list, [spdx-tech@lists.spdx.org](mailto:spdx-tech@lists.spdx.org).
+
+If you would like to work on a fix for any issue, please assign the issue to yourself prior to creating a patch.
+
+## Patches
+
+The source code for `tools-python` is hosted on [github.com/spdx/tools-python](https://github.com/spdx/tools-python). Please review [open pull requests](https://github.com/spdx/tools-python/pulls) before committing time to a substantial revision. Work along similar lines may already be in progress.
+
+To submit a patch via GitHub, fork the repository, create a topic branch from `master` for your work, and send a pull request when your local tests pass (`./setup.py test`).
+
+## Licensing
+
+However you choose to contribute, please confirm in a comment to your pull request that you license your contributions under the terms of [the Developer Certificate of Origin](https://developercertificate.org/).
```

Comment:
The way this should be done is with a `Signed-off-by: name <foo@email.com>`  commit message instead... and this is what is enforced by the bot too, btw ;)

Reply:
Noticed.  Is there a method you'd recommend to locally verify your branch is properly signed-off?  It looks like `rebase --signoff` might be the right way to fix missing signoffs; got an opinion?